### ISSUE TITLE
[Package Issue]: Helm.Helm 3.11.1

### ISSUE THREAD

**Issue by EricStG** [microsoft/winget-pkgs#96973]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

When installing Helm, it sets updates to path to include `%LocalAppData%\Microsoft\WinGet\Packages\Helm.Helm_Microsoft.Winget.Source_8wekyb3d8bbwe`.

Unfortunately, the binaries are under `%LocalAppData%\Microsoft\WinGet\Packages\Helm.Helm_Microsoft.Winget.Source_8wekyb3d8bbwe\windows-amd64`.

### Steps to reproduce

Install Helm
Try to use it

### Actual behavior

Path is not set to the binaries (or the binaries are not in the path)

### Expected behavior

We should be able to run Helm without having to manuallu update the path

### Environment

```shell
Windows: Windows.Desktop v10.0.22621.1194
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.10173.0
```

**Comment by denelon** [comment]
@EricStG you will most likely need to close and reopen your shell/terminal for the Path entry to take effect.

**Comment by EricStG** [comment]
@denelon I did! The path isn't set property to begin with

**Comment by denelon** [comment]
@EricStG I just tried and got the same behavior. I'll dig in and see if I can figure out what is happening.

**Comment by weeix** [comment]
The problem is also reproducible for me. I've been manually adding `\windows-amd64` to the path entry as a workaround.

![Screenshot 2023-02-17 101619](https://user-images.githubusercontent.com/6592791/219541680-1f04d210-c3c6-4323-80db-af9896ac1848.png)

**Comment by denelon** [comment]
https://github.com/microsoft/winget-cli/issues/2909

**Comment by mdanish-kh** [comment]
Resolved in WinGet [v1.5.1081-preview release](https://github.com/microsoft/winget-cli/releases/tag/v1.5.1081-preview)
